script/e2e_test.sh

```diff
@@ -85,4 +85,8 @@ run_test() {
 }
 
 run_e2e_test "conf/server/server.conf"
-run_docker_test "test/configs/server/postgres.conf" "-e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres"
+run_docker_test "test/configs/server/postgres.conf" "-e POSTGRES_PASSWORD=password -p 10864:5432 -d postgres"
+run_docker_test "test/configs/server/mysql.conf" "-e MYSQL_PASSWORD=password -e MYSQL_DATABASE=mysql -e MYSQL_USER=mysql -e MYSQL_RANDOM_ROOT_PASSWORD=yes -p 6612:3306 -d mysql:5.5.62"
```

Comment:
hmm, I wonder if we should be running all these tests against different versions on every CI run? It may be sufficient to only run against the most prevalent version and run the others on merge to master and/or tagged builds (same criteria we use to build images and run more lengthy systems tests)...

Reply:
Yes, I didn't realize that certain tests were only running on merges/tagged branches. I think this should go there and we could also add different versions of Postgres to those tests (later as separate PR).